x-pack/plugins/observability_solution/entity_manager/common/constants_entities.ts

```diff
@@ -5,17 +5,20 @@
  * 2.0.
  */
 
-export const ENTITY_VERSION = 'v1';
-export const ENTITY_BASE_PREFIX = '.entities-observability';
-export const ENTITY_HISTORY_BASE_PREFIX = `${ENTITY_BASE_PREFIX}.history-${ENTITY_VERSION}`;
-export const ENTITY_LATEST_BASE_PREFIX = `${ENTITY_BASE_PREFIX}.latest-${ENTITY_VERSION}`;
-export const ENTITY_HISTORY_TRANSFORM_PREFIX = `entity-history-${ENTITY_VERSION}`;
-export const ENTITY_LATEST_TRANSFORM_PREFIX = `entity-latest-${ENTITY_VERSION}`;
+export const ENTITY_BASE_PREFIX = 'entities';
+export const ENTITY_SCHEMA_VERSION_V1 = 'v1';
```

Comment:
what's the reason to include `V1` in the constant name here? 

it means when we change the value we have to find-replace all the uses of the constant too

Reply:
> it means when we change the value we have to find-replace all the uses of the constant too

Thanks to TypeScript that is a simple rename refactor to affect all files.

> what's the reason to include V1 in the constant name here?

My thinking is that we actually will never change the value of _this_ constant but rather introduce a new constant for `v2` because when we do this, we still need our code to be able to generate the `v1` IDs to continue to manage those definitions that are still using `v1`, so there will be code paths that use the `v1` constants and paths that use the `v2` constants. Hence why I thought that maybe making the schema version an argument to those helpers that generate the IDs.

Happy to undo this change though if that makes more sense, or perhaps include it in **all** other constants 🤷🏼